etc/samples/tendrl_definitions_generic-0.1.sample.yaml

```diff
@@ -0,0 +1,77 @@
+---
+valid_objects:
+- volume
+
+object_details:
+  volume:
+    enabled: True
```

Comment:
What this is meant for and based on what we should set True/False?
Is it like whether this feature is enabled in tendrl??